contracts/abstract/JBOperatable.sol

```diff
@@ -16,7 +17,7 @@ error UNAUTHORIZED();
   Adheres to:
   IJBOperatable: General interface for the methods in this contract that interact with the blockchain's state according to the protocol's rules.
 */
-abstract contract JBOperatable is IJBOperatable {
+abstract contract JBOperatable is ERC165, IJBOperatable {
```

Comment:
apart from the fact that JBController and JBProjects all inherit from this, i dont think there's much value to making this class ERC165. JBProjects might not need it anyways. 
I'd remove here, and add specifically to JBController, keeping JBOperatable single purpose.

Reply:
Done, removed it from the abstract JBOperatable (and interface) and added it to JBController and JBProjects